src/Access/Dialog.vala

```diff
@@ -54,60 +54,56 @@ public class Access.Dialog : Granite.MessageDialog {
     }
 
     construct {
-        skip_taskbar_hint = true;
+        unowned var surface = get_surface ();
+        if (surface is Gdk.X11.Surface) {
+            unowned var x11_surface = (Gdk.X11.Surface) surface;
+            x11_surface.set_skip_taskbar_hint (true);
+        }
+
         resizable = false;
         modal = true;
 
         choices = new List<Choice> ();
-        set_role ("AccessDialog"); // used in Gala.CloseDialog
-        set_keep_above (true);
 
         if (app_id != "") {
             badge_icon = new DesktopAppInfo (app_id + ".desktop").get_icon ();
         }
 
         deny_button = add_button (_("Deny Access"), Gtk.ResponseType.CANCEL) as Gtk.Button;
         grant_button = add_button (_("Grant Access"), Gtk.ResponseType.OK) as Gtk.Button;
-        unowned var grant_context = grant_button.get_style_context ();
 
         if (action == ButtonAction.SUGGESTED) {
-            grant_context.add_class (Gtk.STYLE_CLASS_SUGGESTED_ACTION);
-            set_default (grant_button);
+            grant_button.add_css_class ("suggested-action");
```

Comment:
There's `Granite.STYLE_CLASS_SUGGESTED_ACTION` now, we should use that.

```suggestion
            grant_button.add_css_class (Granite.STYLE_CLASS_SUGGESTED_ACTION);
```

Reply:
Neat, I was hoping there are style constants defined somewhere :muscle: